helm-charts/interoperator/conf/settings.yaml

```diff
@@ -100,6 +100,10 @@ defaults: &defaults
     enabled: {{ .Values.broker.quota.enabled }}
     oauthDomain: {{ .Values.broker.quota.oauthDomain }}
     serviceDomain: {{ .Values.broker.quota.serviceDomain }}
+    mtls:
+      enabled: {{ .Values.broker.quota.mtls.enabled }}
+      cert_secret: {{ .Values.broker.quota.mtls.cert_secret }}
```

Comment:
```suggestion
```
The `cert_secret` is not used inside the pod. So I feel we don't need cert_secret in settings.yaml.